libnavui-androidauto/src/main/java/com/mapbox/androidauto/MapboxCarNavigationManager.kt

```diff
@@ -32,14 +32,12 @@ class MapboxCarNavigationManager(
     }
 
     private var maneuverApi: MapboxManeuverApi? = null
-    private var carDistanceFormatter: CarDistanceFormatter? = null
     private var mapboxNavigation: MapboxNavigation? = null
     private val carTelemetry = MapboxCarTelemetry()
 
     private val routeProgressObserver = RouteProgressObserver { routeProgress ->
-        val distanceFormatter = carDistanceFormatter ?: return@RouteProgressObserver
         val maneuverApi = maneuverApi ?: return@RouteProgressObserver
-        val trip = CarManeuverMapper.from(routeProgress, maneuverApi, distanceFormatter)
+        val trip = CarManeuverMapper.from(routeProgress, maneuverApi, CarDistanceFormatter)
```

Comment:
We can remove `distanceFormatter` parameter in `CarManeuverMapper.from`.

Reply:
Yeah thought about this but was lazy last night. Thanks it's updated